syntax = "proto3";
package proto;
option java_package = "sd.ufc.reserva.model";
option java_outer_classname = "ObjectMessageCallback";

message MessageCallback{
    string mensagem = 1; 
}